Dockerfile

```diff
@@ -1,12 +1,5 @@
-FROM jimmycuadra/rust:1.16.0
+FROM debian:jessie-slim
 
-RUN mkdir -p /rust/app
-WORKDIR /rust/app
+COPY ./target/x86_64-unknown-linux-gnu/release/linkerd-tcp /usr/local/bin/
 
-ENV TARGET=x86_64-unknown-linux-gnu
-
-COPY . /rust/app
-RUN cargo build --release --target=$TARGET
-RUN cp ./target/$TARGET/release/linkerd-tcp /usr/local/bin
-
-ENTRYPOINT ["/usr/local/bin/linkerd-tcp"]
+ENTRYPOINT ["linkerd-tcp"]
```

Comment:
since we're explicitly putting the binary in /usr/local/bin, i'd be inclined to be explicit here so that there's no chance of funnybusiness, i.e. /usr/local/bin-linkerd-tcp